deploy/cr.yaml

```diff
@@ -309,6 +309,18 @@ spec:
 #     - 10.0.0.0/8
 #   serviceAnnotations:
 #     service.beta.kubernetes.io/aws-load-balancer-backend-protocol: http
+  logcollector:
+    enabled: false
+    image: percona/percona-xtradb-cluster-operator:1.6.0-fluentbit
+#    configuration: |
+#      [INPUT]
+#          Name  tail
+#          Path   ${LOG_DATA_DIR}/innobackup.backup.log
+#          Tag   ${POD_NAMESPASE}.${POD_NAME}.innobackup.backup.log
```

Comment:
it is better to add an example for elastic search output

Reply:
done